apps/data/src/pipelines/centrum_pipeline.py

```
@@ -83,6 +104,12 @@
 SERVICE_CREDENTIAL_NAME = spark.conf.get("SERVICE_CREDENTIAL_NAME")
 MONITORING_SLACK_CHANNEL = spark.conf.get("MONITORING_SLACK_CHANNEL")
 
+LARGE_IOT_S3_PATH = spark.conf.get("LARGE_IOT_S3_PATH", "")
```

Ruff (0.15.13)

[error] Undefined name `spark`

(F821)